app/main.py

```diff
@@ -1,8 +1,18 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person["name"], person["age"])
+                   for person in people]
+    for hum in people:
+        if "wife" in hum and hum["wife"]:
+            Person.people[hum["name"]].wife = Person.people[hum["wife"]]
+        if "husband" in hum and hum["husband"]:
+            Person.people[hum["name"]].husband = Person.people[hum["husband"]]
```

Comment:
use get() method instead of parenthesis everywhere for getting value by key to avoid KeyError

Reply:
Here it no needed

Reply:
Because:
1. We already know that "husband" exist in hum
2. And we know that "name" exist also, due to task requirements
https://docs.quantifiedcode.com/python-anti-patterns/correctness/not_using_get_to_return_a_default_value_from_a_dictionary.html?highlight=get